Function_Constructor_Overloading/Level_1/n-nobles.cpp

```diff
@@ -0,0 +1,32 @@
+#include <bits/stdc++.h>
+using namespace std;
+int u,v, q, n, m, o;
+void change(int u){
+
+}
+void change(int u,int v)
```

Comment:
I am afraid, I won't be able to merge programs that just work in ELab, but are not logically sound. Fix these to get it merged!